modules/image_processing/include/image_processing/result_data.h

```diff
@@ -27,6 +27,7 @@ struct ResultData
     ResultData() {}
 
     bool ok;
+    cv::Scalar mean;
```

Comment:
You should also add an include to the opencv header file in this file for this to work. Otherwise including only result_data.h could give an error.

Reply:
Done